cmake/fficonfig.h.in

```diff
@@ -45,98 +45,98 @@
 /* #undef HAVE_AS_SPARC_UA_PCREL */
 
 /* Define if your assembler supports PC relative relocs. */
-#define HAVE_AS_X86_PCREL 1
+#cmakedefine HAVE_AS_X86_PCREL 1
 
 /* Define to 1 if you have the <dlfcn.h> header file. */
-#define HAVE_DLFCN_H 1
+#cmakedefine HAVE_DLFCN_H 1
 
 /* Define if __attribute__((visibility("hidden"))) is supported. */
-#define HAVE_HIDDEN_VISIBILITY_ATTRIBUTE 1
+#cmakedefine HAVE_HIDDEN_VISIBILITY_ATTRIBUTE 1
 
 /* Define to 1 if you have the <inttypes.h> header file. */
-#define HAVE_INTTYPES_H 1
+#cmakedefine HAVE_INTTYPES_H 1
 
 /* Define if you have the long double type and it is bigger than a double */
-#define HAVE_LONG_DOUBLE 1
+#cmakedefine HAVE_LONG_DOUBLE 1
 
 /* Define to 1 if you have the `memcpy' function. */
-#define HAVE_MEMCPY 1
+#cmakedefine HAVE_MEMCPY 1
```

Comment:
Missing test. Could be added before [memmove](https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/blob/5b246c43d012647a194a5b50595dc5968caf13ce/cmake/ConfigureChecks.cmake#L754)